### ISSUE TITLE
str-slice failing when slicing from last character

### ISSUE THREAD

**Issue by HugoGiraudel** [sass/libsass#1123]
```scss
$string: 'Hello world!';

// Expected: last character of string
// Ruby Sass 3.4 [OKAY] returns "!"
// LibSass 3.1   [FAIL] returns ""
.foo {
  content: str-slice($string, str-length($string));
  content: str-slice($string, -1);
}
```

Ref: http://sassmeister.com/gist/622a760397796db2fb74.

**Comment by xzyfer** [comment]
I believe this is a duplicate and fixed in 3.2. Can't link on phone.

**Comment by xzyfer** [comment]
Duplicate https://github.com/sass/libsass/issues/988. Fixed in 3.2.0
